src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/WindowsServiceLifetime.cs

```diff
@@ -84,15 +102,20 @@ public Task StopAsync(CancellationToken cancellationToken)
             return Task.CompletedTask;
         }
 
-        // Called by base.Run when the service is ready to start.
+        /// <summary>
+        /// Called by base.Run when the service is ready to start.
+        /// </summary>
+        /// <param name="args">The command line arguments passed to the service.</param>
         protected override void OnStart(string[] args)
         {
             _delayStart.TrySetResult(null);
             base.OnStart(args);
         }
 
-        // Called by base.Stop. This may be called multiple times by service Stop, ApplicationStopping, and StopAsync.
-        // That's OK because StopApplication uses a CancellationTokenSource and prevents any recursion.
+        /// <summary>
+        /// Called by base.Stop to stop the <see cref="WindowsServiceLifetime"/>.
+        /// </summary>
+        /// <remarks>This may be called multiple times by service Stop, ApplicationStopping, and StopAsync. That's OK because StopApplication uses a CancellationTokenSource and prevents any recursion.</remarks>
```

Comment:
```suggestion
        /// <remarks>This might be called multiple times by service Stop, ApplicationStopping, and StopAsync. That's okay because StopApplication uses a CancellationTokenSource and prevents any recursion.</remarks>
```

Reply:
Ideally those APIs would be `see crefs`.